src/app/features/tech-record/components/tech-record-amend-vrm/tech-record-amend-vrm.component.html

```diff
@@ -5,44 +5,33 @@ <h2 class="govuk-heading-l">{{ makeAndModel }}</h2>
     <div class="govuk-summary-list__row">
       <dt class="govuk-summary-list__key">Vehicle type</dt>
       <dd id="vehicleType" class="govuk-summary-list__value">
-        {{ currentTechRecord?.vehicleType | defaultNullOrEmpty | uppercase }}
+        {{ techRecord?.vehicleType | defaultNullOrEmpty | uppercase }}
       </dd>
     </div>
+
     <div class="govuk-summary-list__row">
       <dt class="govuk-summary-list__key">VRM</dt>
       <app-number-plate *ngIf="vrm" [vrm]="vrm"></app-number-plate>
     </div>
+
     <div class="govuk-summary-list__row">
       <dt class="govuk-summary-list__key">VIN/chassis number</dt>
       <dd id="vehicleType" class="govuk-summary-list__value">{{ vehicle.vin | defaultNullOrEmpty }}</dd>
     </div>
   </dl>
 
   <form [formGroup]="form">
-    <app-text-input
-      #newVrm
-      [label]="form.meta.children![0].label"
-      [formControlName]="form.meta.children![0].name"
-      [width]="width"
-      name="amend-vrm-text"
-      (ngModelChange)="ngOnChanges(); newVrm.value = newVrm.value.toUpperCase()"
-    ></app-text-input>
+    <app-text-input #newVrm formControlName="newVrm" [width]="width"></app-text-input>
```

Comment:
really minor but the name field gives the input an ID, not sure they'll need it for testing as there is only one input on the page